CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+Contributor License
+-------------------
+
+VOLTTRON is an Eclipse Foundation project.  If this is your first time contributing to an Eclipse Foundation project, you'll need to sign the [Eclipse Contributor Agreement][ECA].
+
+- [Create an account](https://dev.eclipse.org/site_login/createaccount.php) on dev.eclipse.org
+- Open your [Account Settings tab](https://dev.eclipse.org/site_login/myaccount.php#open_tab_accountsettings), enter your GitHub ID and click Update Account
+- Read and [sign the ECA](https://dev.eclipse.org/site_login/myaccount.php#open_tab_cla)
+- Use the exact same email address for your Eclipse account and your commit author.
+
+Issues
+------
+
+Search the [issue tracker](https://github.com/volttron/volttron-core/issues) for a relevant issue or create a new one.
+
+Making changes
+--------------
+
+Fork the repository in GitHub and make changes in your fork.
+
+Submit a pull request.
+
+Contact us
+----------
+
+[Join the mailing list][mailing-list] or email volttron@pnnl.gov to discuss your ideas and get help.

```

Comment:
Can we say "Email volttron@pnnl.gov to join the mailing list and discuss your ideas or get help"

Reply:
Also add link to https://github.com/VOLTTRON/volttron-core/discussions

Reply:
We want the user to use the form to ask the questions...we can get rid of the email if you want however.

Reply:
The form is now a link....